client/k8s.go

```diff
@@ -214,7 +214,7 @@ func (k K8sClient) GetCStorVolumeInfoMap(node string) (map[string]*util.Volume,
 		pv, err := k.GetPV(*i.Spec.Source.PersistentVolumeName)
 
 		if err != nil {
-			klog.Errorf("Failed to get PV", i.ObjectMeta.Name)
+			klog.Errorf("Failed to get PV %s", i.ObjectMeta.Name)
```

Comment:
`fmt.printIn` here if we decide to go with it.

Reply:
I had a small discussion with @Abhinandan-Purkait  and we both agree that`fmt.Println` looks better to show errors like `No volumes found`, `No pools found in XYZ namespace`, etc and `klog` looks better for developer facing logs like `timed-out reaching out the k8s-api`, etc. More finer discussion may be suggested by reviewers.